packages/env-graph/src/lib/env-graph.ts

```diff
@@ -166,6 +166,9 @@ export class EnvGraph {
         // process and resolve any other items our env flag depends on
         for (const depKey of envFlagItem.valueResolver?.deps || []) {
```

Comment:
found an edge case in the special envFlag early resolution... Will likely want to clean that up a bit so its not quite as special, but this is ok for now.